src/curated-corpus/components/ScheduleHistory/ScheduleHistory.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { Grid, Typography, Collapse } from '@material-ui/core';
+import { DateTime } from 'luxon';
+import { ApprovedCorpusItemScheduledSurfaceHistory } from '../../../api/generatedTypes';
+import { useStyles } from './ScheduleHistory.styles';
+import {
+  getCuratorNameFromLdap,
+  getScheduledSurfaceName,
+} from '../../helpers/helperFunctions';
+import { Button } from '../../../_shared/components';
+
+interface ScheduleHistory {
+  data: ApprovedCorpusItemScheduledSurfaceHistory[];
+}
+
+export const ScheduleHistory: React.FC<ScheduleHistory> = (
+  props
+): JSX.Element => {
+  const classes = useStyles();
+  const { data } = props;
+
+  const [isShowingHistory, setIsShowingHistory] = useState(false);
+
+  const getDisplayDate = (date: string) => {
+    return DateTime.fromFormat(date, 'yyyy-MM-dd')
+      .setLocale('en')
+      .toLocaleString(DateTime.DATE_FULL);
+  };
```

Comment:
Was thinking about moving this function to the `helperFunctions.ts` file but this function is very specific to this component only. If I were to create a generic `getDisplayDate()` function, it'll have to accommodate for different input types, e.g unix timestamps. So for now I've left it in here.